modules/general/log-analytics/README.md

```diff
@@ -0,0 +1,41 @@
+
+Adds or updates log analytics workspace
+
+## Description
+
+{{ Add detailed description for the module. }}
+
+## Parameters
+
+| Name                                         | Type     | Required | Description                                                                                                                                |
+| :------------------------------------------- | :------: | :------: | :----------------------------------------------------------------------------------------------------------------------------------------- |
+| `name`                                       | `string` | Yes      | The name of the log analytics workspace                                                                                                    |
+| `location`                                   | `string` | Yes      | The location of the log analytics workspace                                                                                                |
+| `skuName`                                    | `string` | Yes      | The SKU of the log analytics workspace                                                                                                     |
+| `dailyQuotaGb`                               | `int`    | Yes      | The daily ingestion quota (in GB) of the log analytics workspace - use "-1" for no limit                                                   |
+| `enableLogAccessUsingOnlyResourcePermisions` | `bool`   | Yes      | When true, the log analytics workspace will only be accessible by using resource permissions                                               |
+| `publicNetworkAccessForIngestion`            | `string` | Yes      | Indicates whether the public network access for ingestion is enabled or disabled                                                           |
+| `publicNetworkAccessForQuery`                | `string` | Yes      | Indicates whether the public network access for query is enabled or disabled                                                               |
+| `tagValues`                                  | `object` | No       | The tag values of the log analytics workspace                                                                                              |
+| `useExisting`                                | `bool`   | No       | When true, the details of an existing log analytics workspace will be returned; When false, the log analytics workspace is created/updated |
+
+## Outputs
+
+| Name              | Type   | Description                                        |
+| :---------------- | :----: | :------------------------------------------------- |
+| workspaceResource | object | An object representing the log analytics workspace |
+| name              | string | The workspace resource name                        |
+| id                | string | The workspace resource ID                          |
+
+## Examples
+
+### Example 1
+
+```bicep
+```
+
+### Example 2
+
+```bicep
+```
```

Comment:
You need to replace these with 1 or more examples. e.g.
 https://github.com/endjin/Endjin.RecommendedPractices.Bicep/blob/d5faa400dbe8f25c33b1319dc3ae4846c7f4cc1f/modules/general/storage-account/README.md?plain=1#L40-L99

Reply:
Added an example in [this](https://github.com/endjin/Endjin.RecommendedPractices.Bicep/pull/41/commits/3a07721932d5db474fa847b63fa1aa236df0d0b6) commit. However, I'm struggling to come up with a meaningful example as I'm not familiar with many of the properties on a `log_analytics_workspace` resource/